metaflow/metaflow_config.py

```diff
@@ -41,9 +41,15 @@ def from_conf(name, default=None):
 ###
 # Default configuration
 ###
+DEFAULT_AUTH = from_conf('METAFLOW_DEFAULT_AUTH', 'aws')
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
+DEFAULT_DATA_PROVIDER = from_conf('METAFLOW_DEFAULT_DATAPROVIDER', '')
```

Comment:
Nit: DEFAULT_DATAPROVIDER so its also consistent with the `from_conf` invocation across other values.

Reply:
Done.